### PR TITLE
Add setup

### PR DESCRIPTION
**Pull request by BasitKorede**
# SETUP FOR PORTFOLIO PROJECT.

### This pull request adds the following.
- It consist of the `Nav-bar` (navigation bar).
- The about me section.
- The language,skill and framework section.
- And various social media platform i can be contacted.

### CODE REVIEW

File: css/index.css [hunk 1]
```diff
@@ -0,0 +1,3 @@
+@import "./reset.css";
+@import "./style.css";
+@import "./screens/large-screen.css" screen and (min-width: 481px);
```
Comment (adejam) [inline css/index.css]: We should not have an import to an empty path

File: index.html [hunk 2]
```diff
@@ -1,18 +1,184 @@
...
+    <link rel="stylesheet" href="./css/index.css" />
+    <title>MY PORTFOLIO</title>
+    <link rel="stylesheet" href="index.css" />
+  </head>
+  <body class="position-relative">
     <header>
-        <h1 class="header">Hello world!</h1>
+      <nav class="display-flex nav-bar justify-content-space-between">
+        <div class="name-div">
+          <a> Basit Korede </a>
```
Comment (adejam) [inline index.html]: This anchor tag has no href attribute. All `<a>` tags should have an HTML attribute. If we don't want to have an empty href then we can use `href="#"` instead

File: index.html [hunk 3]
```diff
@@ -1,18 +1,184 @@
...
+        </div>
+        <button type="button" class="toggle-button">
+          <span class="bar"></span>
+          <span class="bar"></span>
+          <span class="bar"></span>
+        </button>
+      </nav>
```
Comment (adejam) [inline index.html:22]: Only the hamburger button should have the nav tag. And not all the links/elements in the header

File: index.html (hunk 2 above)
Comment (adejam) [inline index.html]: This should have a font size of 18px as described in the design

File: index.html [hunk 4]
```diff
@@ -1,18 +1,184 @@
...
+              in="SourceGraphic"
+              in2="BackgroundImageFix"
+              result="shape"
+            />
+            <feGaussianBlur stdDeviation="25" result="effect1_foregroundBlur" />
+          </filter>
+        </defs>
+      </svg>
+
+      <h1 class="name-header">Basit Korede</h1>
```
Comment (adejam) [inline index.html]: This header should be in the `section`

File: index.html [hunk 5]
```diff
@@ -1,18 +1,184 @@
...
 
-</html>
\ No newline at end of file
+          <li class="social-media-li">
+            <a
+              ><img
+                src="./images/linkdin.svg"
+                class="social-media-image"
+                alt="linkdin icon"
+            /></a>
```
Comment (adejam) [inline index.html:146]: All decorations should always be background images and not  tags. Please refer to this [article](https://www.w3docs.com/snippets/html/when-to-use-html-img-tag-and-css-background-image-property.html) for more on this. Let us fix this in every other case

**Review by adejam — CHANGES_REQUESTED** [review]
Milestone seems to be done with just a few exception. Please let's make changes to the reviews below.

File: css/index.css (hunk 1 above)
Comment (BasitKorede) [reply css/index.css]: That for calling this out, i will make the advice change(s).

File: index.html (hunk 2 above)
Comment (BasitKorede) [reply index.html]: Thanks to you awareness i will make the advice change(s).

File: index.html (hunk 3 above)
Comment (BasitKorede) [reply index.html:22]: thanks for pointing this out, it so bad to have made this type of mistake. I will make the due change(s).

**Review by adejam — APPROVED** [review]
Looks good to me. We should merge this PR and move to the next milestone.

**Pull request merged**